tests/components/reolink/test_sensor.py

```diff
@@ -0,0 +1,105 @@
+"""Test the Reolink sensor platform."""
+
+from unittest.mock import MagicMock, patch
+
+from freezegun.api import FrozenDateTimeFactory
+
+from homeassistant.components.reolink import DEVICE_UPDATE_INTERVAL, const
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import STATE_UNAVAILABLE, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from .conftest import TEST_NVR_NAME, TEST_UID
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+
+async def test_sensors(
```

Comment:
Consider using snapshots to fixate all the sensor entities and their state

Reply:
I like the current approach better since it will not fail these tests if I simply a a new sensor entity.
Later once I achieved 100% coverage, I can add some snapshot tests to fixate all entities.